src/gui-wizard-gtk/wizard.c

```diff
@@ -2004,13 +1980,11 @@ static gboolean consume_cmd_output(GIOChannel *source, GIOCondition condition, g
             dd = dd_opendir(g_dump_dir_name, DD_OPEN_READONLY | DD_FAIL_QUIETLY_EACCES);
         if (!dd)
             xfunc_die();
-        char *not_reportable = dd_load_text_ext(dd, FILENAME_NOT_REPORTABLE, 0
-                                            | DD_LOAD_TEXT_RETURN_NULL_ON_FAILURE
-                                            | DD_FAIL_QUIETLY_ENOENT
-                                            | DD_FAIL_QUIETLY_EACCES);
+        g_autofree char *not_reportable = dd_load_text_ext(dd, FILENAME_NOT_REPORTABLE,
+                DD_LOAD_TEXT_RETURN_NULL_ON_FAILURE | DD_FAIL_QUIETLY_ENOENT
+                | DD_FAIL_QUIETLY_EACCES);
```

Comment:
Ouch, this really is less readable. Can you parenthesize the ORing or add newlines as previously?

Reply:
Will try.